src/components/Create/components/pages/ReviewDeploy/components/DeploySuccess/DeploySuccess.tsx

```diff
@@ -48,37 +49,15 @@ export const DeploySuccess = ({ projectId }: { projectId: number }) => {
 
   return (
     <div
-      style={{
-        marginTop: isMobile ? '1rem' : '7rem',
-        display: 'flex',
-        flexDirection: 'column',
-        alignItems: 'center',
-        justifyContent: 'center',
-        textAlign: 'center',
-      }}
+      className={classNames(
+        'flex flex-col items-center justify-center text-center',
+        isMobile ? 'mt-4' : 'mt-28',
```

Comment:
its gonna be hectic when we can bump the `isMobile` in favor of legit css media queries.

In fact...could we start this right now? `md:mt-4` and such